nbqa/__main__.py

```diff
@@ -516,7 +544,9 @@ def _run_on_one_root_dir(
             for notebook in _get_all_notebooks(cli_args.root_dirs)
         }
 
-        _preserve_config_files(configs.nbqa_config, tmpdirname, project_root)
+        _preserve_config_files(
+            cli_args.command, configs.nbqa_config, tmpdirname, project_root
```

Comment:
Would it be better to write it this way? 

```
config_files = (configs.nbqa_config and [configs.nbqa_config]) or CONFIG_FILES[cli_args.command]
_preserve_config_files(
      config_files, tmpdirname, project_root
```
Currently `_preserve_config_files` is looking like find configuration files and then preserve those. By passing the configuration files, `_preserve_config_files` this function is kind of simplified

Reply:
that's better, thanks!